subsys/fs/nvs/nvs_priv.h

```diff
@@ -11,9 +11,9 @@
 extern "C" {
 #endif
 
-#define SYS_LOG_DOMAIN "fs/nvs"
+#define LOG_MODULE_NAME fs_nvs
```

Comment:
i would move this block to nvs.c next to module registration.
Add:
`#define LOG_LEVEL CONFIG_NVS_LOG_LEVEL`
To get logging configuration from kconfig.